common/test/acceptance/tests/lms/test_bookmarks.py

```diff
@@ -310,8 +318,7 @@ def test_bookmarks_list(self):
         """
```

Comment:
Search for `click on Bookmarks button` and see if it is relevant anywhere.  This one should be changed.  Maybe you could add this comment to the single navigation test?